src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/cachemanager/ImmutableComputableGraphUtils.java

```diff
@@ -0,0 +1,115 @@
+package org.broadinstitute.hellbender.tools.coveragemodel.cachemanager;
+
+import org.broadinstitute.hellbender.utils.Utils;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * A utilities class for {@link ImmutableComputableGraph}.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class ImmutableComputableGraphUtils {
```

Comment:
That would restrict access to only classes in `org.broadinstitute.hellbender.tools.coveragemodel.cachemanager` and classes in `org.broadinstitute.hellbender.tools.coveragemodel` will not have access to builder methods, right?